api/src/apps/parkeergarages/models.py

```diff
@@ -8,17 +35,16 @@ class ParkingLocation(models.Model):
     type = models.CharField(max_length=20)
     geometrie = models.PointField(name="geometrie", srid=4326, null=True)
     state = models.CharField(max_length=10)
-    free_space_short = models.IntegerField()
-    free_space_long = models.IntegerField()
-    short_capacity = models.IntegerField()
-    long_capacity = models.IntegerField()
+    free_space_short = models.IntegerField(blank=True, null=True, default=None)
+    free_space_long = models.IntegerField(blank=True, null=True, default=None)
+    short_capacity = models.IntegerField(blank=True, null=True, default=None)
+    long_capacity = models.IntegerField(blank=True, null=True, default=None)
```

Comment:
Maybe add help test "Total number of long duration parking spots" (also for the other 3)